TestPlaybooks/playbook-Maltiverse_Test.yml

```diff
@@ -0,0 +1,580 @@
+id: Maltiverse Test
+version: 11
```

Comment:
Change this to `-1`. It's how we lock out versions and prevent version conflicts.